tests/test-image-deblurring.cpp

```diff
@@ -0,0 +1,133 @@
+/*
+ * test-image-deblurring.cpp - test image deblurring
+ *
+ *  Copyright (c) 2017 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Author: Andrey Parfenov <a1994ndrey@gmail.com>
+ * Author: Wind Yuan <feng.yuan@intel.com>
+ */
+
+#include "test_common.h"
+#include "test_inline.h"
+#include <unistd.h>
+#include <getopt.h>
+#include "ocl/cl_device.h"
+#include "ocl/cl_context.h"
+#include "ocl/cl_blender.h"
+#include "image_file_handle.h"
+#include "ocl/cv_image_deblurring.h"
+#include <opencv2/opencv.hpp>
+#include <opencv2/core/ocl.hpp>
+
+using namespace XCam;
+
+void
+init_opencv_ocl (SmartPtr<CLContext> context)
+{
+    cl_platform_id platform_id = CLDevice::instance()->get_platform_id ();
+    char *platform_name = CLDevice::instance()->get_platform_name ();
+    cl_device_id device_id = CLDevice::instance()->get_device_id ();
+    cl_context context_id = context->get_context_id ();
+    cv::ocl::attachContext (platform_name, platform_id, context_id, device_id);
+}
+
+static void
+usage(const char* arg0)
+{
+    printf ("Usage:\n"
+            "\t--input,    input image(RGB)\n"
```

Comment:
There's a warning in compiling.
you may miss a line
 "%s --input image --output image\n"